forest/daemon/src/daemon.rs

```diff
@@ -255,6 +256,8 @@ pub(super) async fn start(opts: CliOpts, config: Config) -> anyhow::Result<Rolli
         false
     };
 
+    load_bundles(epoch, &config, db.clone()).await?;
```

Comment:
Does this load bundles into db each time we run the node? Looks like it does to me.

Reply:
Yes, but given it's a KV DB it shouldn't be an issue.